Tutorial.md

```diff
@@ -0,0 +1,66 @@
+# MSON Tutorial
+
+MSON is short for Markdown Syntax for Object Notation, and it's a way to represent data structures in a human readable plain text form. This tutorial will take you though the basics of MSON and how you can use MSON to describe a data structure.
+
+> **NOTE:** **Additional MSON Resources**
+>
+> + [MSON Specification](https://github.com/apiaryio/mson/blob/master/MSON%20Specification.md)
+> + [Examples](https://github.com/apiaryio/mson#notational-conventions)
+
+## Data structures in MSON
+
+To get started, we're going to create a data structure in MSON to represent a Person which has a name.
+
+A data structure to store a Person could be represented in MSON using the following:
+
+```apib
+# Person
+
++ name
+```
+
+Here we've declared our own named type called "Person". This is indicated by using a heading. This is one or more `#` symbols followed by a string representing the name of our data structure. By default, named types like `Person` are object types.
```

Comment:
2x "This is ... " I'd suggest change "This is one or more ..." to "A heading is one or more..."